test/test_suite_test.go

```diff
@@ -56,8 +56,14 @@ func init() {
 		getOrSetEnvVar(k, v)
 	}
 
+	// Only export data to Prometheus in case that is ran on Jenkins
+	if helpers.IsRunningOnJenkins() {
+		config.PrometheusEnabled = true
```

Comment:
What would happen if I ran this locally? Could I export whatever I wanted to Prometheus?

Reply:
Why do you want to export data to Prometheus?

Reply:
I will rephrase; it seems like anyone can export any data they want to Prometheus. Shouldn't this be authenticated in some way?

Reply:
Dicussed offline.